Already on 'master'
Client-Request-ID=a2ffef15-6e38-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:03:36 GMT, Server-Request-ID=c9c29b74-e01e-00d1-4445-02e96e000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:c9c29b74-e01e-00d1-4445-02e96e000000Time:2018-06-12T12:03:37.3037127Z</Message></Error>.
Client-Request-ID=a2ffef17-6e38-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:03:36 GMT, Server-Request-ID=404fa5d1-001e-00f2-1145-0286a5000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:404fa5d1-001e-00f2-1145-0286a5000000Time:2018-06-12T12:03:37.4772093Z</Message></Error>.
Client-Request-ID=a2ffef19-6e38-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:03:36 GMT, Server-Request-ID=bfe78711-f01e-00ec-0445-025c48000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:bfe78711-f01e-00ec-0445-025c48000000Time:2018-06-12T12:03:37.6670089Z</Message></Error>.
Client-Request-ID=a2ffef1b-6e38-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:03:37 GMT, Server-Request-ID=2427f5e9-c01e-00c6-1f45-02290d000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:2427f5e9-c01e-00c6-1f45-02290d000000Time:2018-06-12T12:03:37.8404560Z</Message></Error>.
Traceback (most recent call last):
  File "./02.local.py", line 92, in <module>
    min(metrics['mse']), 
KeyError: 'mse'
Client-Request-ID=91eb6c8b-6e3d-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:38:56 GMT, Server-Request-ID=838ee9c1-801e-00ac-784a-0275a6000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:838ee9c1-801e-00ac-784a-0275a6000000Time:2018-06-12T12:38:56.8655265Z</Message></Error>.
Client-Request-ID=91eb6c8d-6e3d-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:38:56 GMT, Server-Request-ID=47aba4f0-201e-00aa-1a4a-0282de000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:47aba4f0-201e-00aa-1a4a-0282de000000Time:2018-06-12T12:38:57.0392966Z</Message></Error>.
Client-Request-ID=92d92fed-6e3d-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:38:56 GMT, Server-Request-ID=eaf9739d-601e-0043-484a-027ed8000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:eaf9739d-601e-0043-484a-027ed8000000Time:2018-06-12T12:38:57.2569735Z</Message></Error>.
Client-Request-ID=92d92fef-6e3d-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 12:38:56 GMT, Server-Request-ID=67803f28-a01e-007c-3b4a-02c904000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:67803f28-a01e-007c-3b4a-02c904000000Time:2018-06-12T12:38:57.4301416Z</Message></Error>.
Traceback (most recent call last):
  File "./10.o16n.py", line 88, in <module>
    model.delete()
  File "/home/ninghai/miniconda3/envs/preview/lib/python3.6/site-packages/azureml/core/model.py", line 313, in delete
    'Content: {}'.format(resp.status_code, resp.headers, resp.content))
azureml.exceptions.azureml_exception.WebserviceException: Received bad response from Model Management Service:
Response Code: 412
Headers: {'Date': 'Tue, 12 Jun 2018 12:46:46 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'api-supported-versions': '2018-03-01-preview', 'x-ms-client-request-id': '730a12427ca0432984124ef133a56b38', 'x-ms-client-session-id': '', 'Strict-Transport-Security': 'max-age=15724800; includeSubDomains; preload'}
Content: b'{"code":"PreconditionFailed","statusCode":412,"message":"PreconditionFailed","details":[{"code":"DeletionRequired","message":"The model with id best_model.pkl:189 can not be deleted because it is currently being used in one or more images"}]}'
Already on 'master'
